css/style.css

```diff
@@ -0,0 +1,918 @@
+}
+
+.paragraph img {
+  width: 31px;
+  height: auto;
+}
+
+.starz img {
+  width: 35px;
+}
+
+.app-img img {
+  width: 95%;
+}
+
+.right-img img {
+  width: 100px;
+  height: 90px;
+}
+
+.art-foot li img {
+  width: 12px;
+  height: 10px;
+}
+
+.bottom-context p {
+  text-align: center;
+}
+
+.latest-news a {
+  display: flex;
+  text-decoration: none;
+  color: red;
+  align-items: center;
+  justify-content: space-between;
+  width: 95%;
+}
+
+.fund-cap,
+.funding-bottom a {
+  text-decoration: none;
+}
+
+.funding-bottom {
+  width: 95%;
+  display: flex;
+  flex-direction: column;
+  padding: 10px 0 10px 5px;
+  border-bottom: 0.5px solid rgb(209, 206, 206);
+}
+
+.funding-link a {
+  text-decoration: none;
+  color: #f22;
+  padding: 2px 20px;
+  border-right: 2px solid red;
+}
+
+.atics-1 a {
+  text-decoration: none;
+  color: #f22;
+  font-size: x-large;
+  padding: 10px 5px 5px 0;
+}
+
+.atics-1 a h3 {
+  margin-left: 15px;
+}
+
+.header-funding h2 a {
+  color: #f22;
+  text-decoration: none;
+  padding: 20px;
+  line-height: 18px;
+}
+
+.art-foot li a {
+  color: #eee;
+  padding-left: 10px;
+}
+
+.art3 ul {
+  padding-top: 85px;
+}
+
+.latest-news {
+  padding-left: 1.5%;
+  margin-top: 1rem;
+}
+
+.caption-img p {
+  color: #888;
+  padding-top: 10px;
+}
+
+.fund-cap p {
+  color: #887c7a;
+  font-size: 14px;
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.latest-news a p {
+  float: inline-end;
+  padding-right: 30px;
+}
+
+.latest-news h2 {
+  color: red;
+  font-size: 24px;
+  font-weight: bold;
+  padding-left: 30px;
+}
+
+.news1 {
+  margin-top: 30px;
+  display: flex;
+  flex-flow: row;
+}
+
+.news {
+  display: flex;
+  flex-flow: column;
+}
+
+.caption-img {
+  margin-left: 10px;
+}
+
+.caption-img h4 {
+  color: #000;
+}
+
+.article-2 {
+  display: flex;
+  flex-direction: column;
+  padding-left: 1.5%;
+  margin-top: 40px;
+}
+
+.red-a {
+  text-decoration: none;
+}
+
+.funding-expand {
+  padding-left: 1.5%;
+  margin-top: 5rem;
+  display: flex;
+  flex-flow: column;
+}
+
+.header-funding {
+  font-size: x-large;
+}
+
+.funding-link {
+  display: flex;
+  flex-flow: column;
+}
+
+.funding-grid {
+  position: relative;
+  margin-top: 15px;
+}
+
+.red-img {
+  position: absolute;
+  width: 60px;
+  height: 60px;
+  top: 10px;
+  left: 10px;
+  z-index: 1;
+}
+
+.red-sec {
+  position: relative;
+  background-color: #f42;
+  width: 97%;
+  height: 40px;
+}
+
+.red-sec h4 {
+  color: white;
+  padding: 15px 0 0 80px;
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.below-red {
+  position: relative;
+  background-color: #fde7e5;
+  width: 97%;
+  height: 60px;
+}
+
+.below-red h4 {
+  color: black;
+  padding: 15px 0 0 80px;
+  font-weight: 700;
+}
+
+.below-red h5 {
+  padding: 5px 0 0 80px;
+  font-size: 20px;
+  font-size: 14px;
+  color: #887c7a;
+}
+
+.fund-cap {
+  padding: 15px 0 5px 10px;
+}
+
+.grid-2 {
+  display: flex;
+  flex-flow: column;
+}
+
+.grid-3 {
+  display: flex;
+  flex-flow: column;
+}
```

Comment:
Don't repeat yourself. In order to minimize code, you can group selectors with the same style definition.